bmc/connection_test.go

```diff
@@ -66,6 +67,8 @@ func TestOpenConnectionFromInterfaces(t *testing.T) {
 
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
+			defer goleak.VerifyNone(t)
```

Comment:
Based on the goleak docs, couldn't this just be defined at the top in the test function, instead of being part of the loop?

Reply:
It could be, but then how would you know which sub test resulted in the leak? It's akin to putting it in TestMain, we wouldn't know what test triggered the leak.